site/public/js/rainbow-customization.js

```diff
@@ -596,6 +664,16 @@ function setCustomizationItemVisibility(elem) {
     else {
         $(cust_item_id).hide();
     }
+
+    // manual grading is dependent on final grade cutoffs
+    if (checkbox_name === 'final_grade') {
+        if (is_checked) {
+            $('#manual-grading').show();
+        }
+        else {
+            $('#manual-grading').hide();
+        }
```

Comment:
This could be simplified with jQuery's [`.toggle(bool)`](https://api.jquery.com/toggle/#toggle-display) method.

Reply:
It needs the `.show()`/`.hide()` to load the page properly, since checkboxes are selected on page load based on the JSON.

Reply:
You can pass a boolean to `.toggle()` to indicate the status it should toggle to.

Reply:
Changed in [ced8003](https://github.com/Submitty/Submitty/pull/10653/commits/ced80036145a5effcf9b52334bcf5d16f95aa506)